src/pages/AboutUs.tsx

```diff
@@ -144,9 +144,8 @@ const AboutUs = (): JSX.Element => {
                     sub="Assistant Professor at Simon Fraser University, Canada Research Chair"
                     avatarSrc="https://www.sfu.ca/content/sfu/physics/people/faculty/hoikwanl.img.1629284380.png"
                     description={
-                        "My research aims to solve the current practicality issues of quantum technology. " +
-                        "I will likely be facilitated by the freedom of Canada, friendliness of Canadians, " +
-                        "and fragrance of Tim Hortons coffee."
+                        "I aim at solving practicality issues of quantum technology, likely to be aided by the" +
```

Comment:
"I aim _to solve_ practicality issues..."

Reply:
```suggestion
                        "I aim to solve practicality issues of quantum technology, likely to be aided by the" +
```

Reply:
Though I think either is fine https://ludwig.guru/s/aim+at+solving